elasticsearch_service/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"internship_project/elasticsearch_service/helpers"
+	"sync"
+)
+
+var (
+	r  map[string]interface{}
+	wg sync.WaitGroup
+)
+
+func main() {
```

Comment:
I suggest moving everything kafka and elasticsearch related to the main project, not running it as a separate one. In main you can register all services and clients, no need for separate projects. It will be easier to maintain and test.

Reply:
Done.